ci-operator/step-registry/openshift-qe/workers-infra-workload/openshift-qe-workers-infra-workload-commands.sh

```diff
@@ -183,7 +183,7 @@ function create_machineset() {
 
     #Set default value for key VARIABLE
     #Use the first machineset name by default if no REF_MACHINESET_NAME specified
-    ref_machineset_name=$(oc -n openshift-machine-api get -o 'jsonpath={range .items[*]}{.metadata.name}{"\n"}{end}' machinesets.m | grep worker | grep -v rhel | head -n1)
+    ref_machineset_name=$(oc -n openshift-machine-api get -o 'jsonpath={range .items[*]}{.metadata.name}{"\n"}{end}' machinesets.m | grep worker | head -n1)
```

Comment:
@SachinNinganure Let's don't do the 2 changes in this file as Liquan suggested, this is for the machineset's name, not the label.